xDrip/Controllers/KeepAliveController.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  KeepAliveController.swift
+//  xDrip
+//
+//  Created by Ivan Skoryk on 03.07.2020.
+//  Copyright © 2020 Faifly. All rights reserved.
+//
+
+import Foundation
+import AVKit
+
+final class KeepAliveController: NSObject {
+    static let shared = KeepAliveController()
```

Comment:
Why does it have to be a singleton? It's not accessed across the app. Just create an instance of it in launch controller.